### DISCUSSION TITLE
Equipo 13 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#19] [Proyecto Final]
Hola @NancyAurora,
Aquí está el hilo en el foro para tu equipo. Podrías conversar aquí sobre tus avances con el proyecto y avisar al Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengas al respecto.
Con la emoción de seguir el desarrollo de tu proyecto,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @NancyAurora 

Gracias por entregar las etapas 0 y 1 del proyecto final. Las he revisado y te escribo para felicitarte por haber completado el trabajo, por haber hecho dos entrevistas, y también para compartir retroalimentación y sugerencias contigo.

**Sobre el protocolo de la entrevista:**
- Podría ser una buena idea empezar con algo para romper el hielo, ej., pregunta / broma / historia / etcetera
- Para la pregunta 2, para profundizarla un poquito, podría preguntar ¿qué causa tu opinion sobre el nivel de sostenibilidad de la moda?

¡Una buena idea crear una mapa de afinidad! 
**Sobre la mapa:**
- En la última columna, el subtitulo dice "Customer Action" pero en las demás columnas los subtítulos son por persona ... ¿hay manera de crear subtítulos consistentes? O ¿agregar una explicación de porque cambió el formato?
- ¿Has pensado en crear mapas de empatía, una por persona? Puede ser una manera eficiente para visualizar diferencias entre las experiencias de los entrevistados.

**Sobre la etapa 2:** 

Para la etapa 2, una sugerencia de mi parte sería hacer la actividad 8S (Ocho Loco) individualmente y luego usar SCAMPER (u otra estrategias en los videos) para ordenar las ideas y elegir una sola idea con que trabajarán en la etapa 3. (Es decir, repetir el proceso que practicamos durante las sesiones interactivas esta semana.)

¿Tienes preguntas para mí? Cualquier cosa, aquí ando.

Mucha suerte con la etapa 2,
Kasturi

**Comment by NancyAurora** [comment]
Hola Kasturi ¿Qué tal?
Muchas gracias por la felicitación y por la retroalimentación.
Te comento, en la etapa 0, realicé una modificación para romper el hielo y
modifique mi pregunta 2 (ahora 3) como me lo mencionaste; con respecto al
mapa quite el subtitulo porque se me pasó modificar la plantilla
predeterminada con la que estaba trabajando y por eso eso decía "Customer
Action" (seré más cautelosa a la hora de entregar mis actividades
siguientes)
Por otro lado creo que el mapa de afinidad me permite visualizar las
diferencias entre las experiencias de las personas entrevistadas de una
mejor manera y me es más fácil de comprender y ordenar la variedad de
ideas, y en el caso del mapa de empatía me llego a confundir un poco.

Y bueno para finalizar me gustaría solicitar una prórroga (de nuevo) para
la etapa 2, lo que pasa es que estoy en el proceso de admisión para entrar
a la universidad y me tiene muy ocupada con todos los requisitos que tengo
que entregar y lo que tengo que estudiar para el examen de admisión, por
favor.

**Reply by KasturiSShah** [reply]
Hola @NancyAurora 
Estoy muy bien gracias, ¿cómo estás?
Gracias por actualizar las etapas 0 y 1. He visto los cambios, y me parecen excelentes. Si la mapa de afinidad fuera más intuitiva para ti, sería mejor solo usarla. Para que puedas consultarla en próximas etapas y ir desarrollando el proyecto.
Claro que sí, tienes prórroga para la etapa 2 hasta este domingo. ¿Sería suficiente tiempo?
Mucha suerte con los exámenes de admisión y los formularios etc que tienes que entregar. Entiendo completamente, este proceso es muy importante. Te recomiendo que conversemos para ajustar cualquier fecha límite para cualquier etapa del proyecto final para que puedas priorizar las otras tareas que tienes.
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @NancyAurora 

Espero estés muy bien y que pasaste una linda semana pasada durante el feriado.

Gracias por entregar la etapa 2 del proyecto final. La he revisado y te escribo con retroalimentación. En primer lugar, me encantó el diseño gráfico que usaste para presentar tus ideas generadas en 8S. Las ideas son claras, concisas, creativas, y bien definidas, ¡bien hecha!

Algunas sugerencias/preguntas de mi parte:
1. Recomiendo que usas SCAMPER para reducir y organizar tus ideas generadas en 8S. SCAMPER te ayudará a identificar 1 o 2 ideas con que trabajarás en la etapa 3 para crear un prototipado low fidelity. Lo probarás en la etapa 4. 
2. ¿Vas a grabar un vídeo explicando tu proceso de generación de ideas y uso de SCAMPER? El vídeo me ayudará a aprender sobre de proceso de ideación. Si no tuvieras tiempo para grabarlo, me avisarías y discutiríamos juntas cómo avanzarías.
3. Para la etapa 3, ¿hasta que fecha quisieras una prórroga? Me avisas para que pueda actualizar mi lista.

Mucha suerte con la etapa 3 del proyecto.

Saludos a la distancia,
Kasturi

**Comment by NancyAurora** [comment]
Hola Kasturi, espero te encuentres muy bien, me alegra saber que te gusto
el gráfico y las ideas del mismo, bueno para empezar lamento la demora en
la entrega de cada etapa; pero el tiempo me absorbe; sin embargo
entregaré cada actividad (aunque tarde un poquito más a comparación de los
otros equipos), ya estoy haciendo el SCAMPER para proceder y grabar el
video, aunque quizá no logre hacer los cinco minutos me esforzare para que
quede todo muy claro.

Espero terminar lo que me falta entre hoy y mañana, cuando esté listo te
enviare un correo para que lo revises y me des las observaciones
pertinentes, muchas gracias.
Deseo que tengas una linda semana,
Nancy.

**Comment by NancyAurora** [comment]
Hola Kasturi, ya subí la etapa 2, pero tuve un problema, grabe el video
varias veces pero no entiendo porque se escucha un ruido de fondo (que
lastima al oído) intente grabar con diferentes medios pero aún así se
escuchaba el mismo ruido, te dejo el Link por aquí, aun así traté que en
las imágenes todo estuviera bien detallado
https://vimeo.com/817139933/820978693d

**Reply by KasturiSShah** [reply]
Hola @NancyAurora 
Gracias por el link al vídeo, no te preocupes el ruido. He insertado la liga al vídeo a tu etapa 2 [aquí](https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-13/etapa-2/README.md) 
Si te parece mejor no insertarlo ahí, siempre puedes editar el archivo para quitar la liga.
Kasturi

**Comment by KasturiSShah** [comment]
Hola @NancyAurora 

Gracias por completar la etapa 2 del proyecto final. Me gustó mucho el diseño gráfico que usaste para presentar tus ideas 8S, para explicar el método SCAMPER, y para explicar los componentes principales de tu idea. 

Ya es momento para hacer prototipado de un aspecto de tu proyecto final. Por ejemplo, puede ser una diagrama de flujo que muestra cada aspecto de tu idea -- desde los contenedores de textiles hasta el foro. La otra opción sería enfocarte en un aspecto solamente, por ejemplo, el foro o el bazaar, y crear un dibujo de dicho aspecto. _Para el prototipo baja fidelidad, es importante destacar que solo tienes que crear un dibujo (en papel y lapiz está bien)_

Para la etapa 4, la tarea es mostrar el prototipo a un usuario, recibir sus comentarios/retroalimentación, y actualizar el prototipo según esta conversación para crear un prototipo alta fidelidad.

Para la etapa 5, la tarea es crear una plantilla Sprint en la cual identificarás algunas tareas que quisieras hacer en el futuro (plantilla [aquí](https://docs.google.com/spreadsheets/d/1aboqGYEJvPv77T5M_uEyUy83oAVKXS2J1Jxi2_48vLQ/copy?usp=sharing)). Te comento que no es necesario completar las etapas 3 y 4 para entregar la etapa 5, así que puedes pasar corto tiempo (30 minutos) en la etapa 5 (es corta), entregarla, y luego avanzar con las etapas 3 y 4.

Finalmente ..
¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar tu presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si te conviene la asignación de tu equipo a una de las dos sesiones de clausura del curso, o si quieres cambiar su día asignado al otro día. (Horario disponible [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing).)

_No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo._

Estoy aquí para apoyarte, déjame saber cualquier pregunta sobre el proyecto.

Mucha suerte, mucho éxito, feliz fin de semana,
Kasturi

**Comment by NancyAurora** [comment]
Hola Kasturi, espero te encuentre muy bien.
Muchas gracias por la retroalimentación.
El día de mañana me toca exponer pero no podré asistir, me cambiaron la
fecha para una entrevista por parte de la universidad en la que estoy
solicitando una beca (me acaban de avisar, por eso hasta ahorita te estoy
avisando)
Entonces tengo que asistir me toca a las 10:30 am, el problema es que tengo
que tomar diferentes medios de transporte para llegar al campus y no podré
conectarme a la sesión para presentar un proyecto, lo siento mucho.

**Reply by KasturiSShah** [reply]
Hola @NancyAurora 
Gracias por avisarme 😄 
En primer lugar, ¡felicidades por la entrevista! Y mucha suerte.
Te animo a terminar las etapas del proyecto final. Siento que tienes una estética muy linda para el diseño, lo veo claramente en cada etapa que completas. Creo que te podría servir tener un proyecto así completado...puede abrir otras oportunidades, por eso, te animo a terminarlo.
Mucha pero mucha suerte!
Kasturi

**Comment by NancyAurora** [comment]
Hola Kasturi, espero te encuentres muy bien.
Primero que nada quiero agradecer tu comprensión.
Por otro lado, ¡claro! que terminaré el proyecto, de hecho estoy aplicando
en la universidad para la carrera de diseño industrial y este curso me ha
dado un panorama sobre el diseño muy interesante y tus palabras me han
hecho sentir feliz.
Aunque diseñe muy rápido las diapositivas que tenía planeadas para la
presentación del día sábado me tomaré un tiempo para hacer algo mejor y
poder compartirlo con el equipo de Patrones Hermosos, realmente estoy muy
motivada a concluirlo aunque vaya un poco despacio debido a los otros
requisitos de otras universidades que necesito completar y absorben mi
tiempo.

Espero que tengas una linda semana.

Nancy.

**Comment by NancyAurora** [comment]
Hola Kasturi, espero te encuentres muy bien, queria contarte, que logre ser
admitida en el Tecnológico de Monterrey con una beca del 100%, actualmente
me encuentro en la entrada de estudios creativos, para poder estudiar
diseño, debo agradecer mucho al curso de Design Thinking porque he empleado
mucho de lo que aprendí. en mis proyectos.
La finalidad de este correo es solicitar permiso a ti y a todo el equipo de
Patrones hermosos, para poder promocionar la beca de Líderes del Mañana,
otorgada por el Tecnológico de Monterrey y así poder dar a conocer esta
oportunidad a todas las chicas e invitarlas a participar.

Quedo al tanto de tu respuesta.

Nancy.
